From: Sebastien Bacher <seb128@ubuntu.com>
Subject: dpkg also supports *.ddeb
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/bash-completion/+bug/568404
Origin: vendor
Forwarded: no

---
 completions/dpkg |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- bash-completion.orig/completions/dpkg
+++ bash-completion/completions/dpkg
@@ -55,7 +55,7 @@ _dpkg()
         -c|-i|-A|-I|-f|-e|-x|-X|-W|--install|--unpack|--record-avail| \
         --contents|--info|--fsys-tarfile|--field|--control|--extract| \
         --show)
-            _filedir '?(u)deb'
+            _filedir '?(u|d)deb'
             return 0
             ;;
         -b|--build)
