From: Tomas Janousek <tomi@nomi.cz>
Subject: fix version detection for Debian's GCC
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=665651
Origin: vendor, https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=665651
Forwarded: no

---
 completions/gcc |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- bash-completion.orig/completions/gcc
+++ bash-completion/completions/gcc
@@ -47,8 +47,8 @@ _gcc()
 } &&
 complete -F _gcc gcc g++ g77 gcj gpc &&
 {
-    cc  --version 2>/dev/null | grep -q GCC && complete -F _gcc cc  || :
-    c++ --version 2>/dev/null | grep -q GCC && complete -F _gcc c++ || :
+    cc  --version 2>/dev/null | grep -q 'GCC\|Debian' && complete -F _gcc cc  || :
+    c++ --version 2>/dev/null | grep -q 'GCC\|Debian' && complete -F _gcc c++ || :
 }
 
 # ex: ts=4 sw=4 et filetype=sh
